### PR TITLE
Reflow flyouts when zoom level changes to keep block zoom level in sync.

### PR DESCRIPTION
**Pull request by gonfunko**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves
https://github.com/google/blockly/issues/4931

### Proposed Changes
Updates flyouts to listen for zoom events and re-render themselves so that their blocks reflect the new zoom level.

#### Behavior Before Change
Zoom level changes are not reflected in an open flyout until a click occurs inside the flyout itself.

#### Behavior After Change
Zoom level changes are immediately reflected in an open flyout.

### Test Coverage
Manually verified and ensured all test suites pass.

### CODE REVIEW

**Review by alschmiedt — COMMENTED** [review]
So I think this might be even simpler than this change. 

We use flyouts in two different ways on the workspace: 
- When we have a category toolbox we have flyouts that the toolbox owns. `(workspace.getToolbox().getFlyout()`
- When we have a "simple toolbox" (a flyout that sits on the workspace always open) then the workspace owns the flyout. `workspace.flyout_`

We actually do the reflow for the flyouts that live on the workspace on [this line](https://github.com/google/blockly/blob/fc6a1be3d982349292b630d2818b3b5509688d5f/core/workspace_svg.js#L2213). I think we could leverage this for flyouts that are open as well by using `workspace.getFlyout()` instead of `workspace.flyout_` and checking that the flyout `isVisible` before doing the reflow.

How we deal with the two kinds of flyouts is not very clear so let me know if you have any questions!

**Comment by gonfunko** [comment]
Oh nice, good call! That appears to work fine too, and is a lot simpler. Updated the PR accordingly - thanks!

**Review by alschmiedt — APPROVED** [review]
Awesome, thanks!

Approved, but just a reminder to wait until after release to merge in : )

**Pull request merged**
